src/components/comment-form/comment-form.scss

```diff
@@ -18,3 +18,9 @@
     grid-column: span 1;
   }
 }
+
+@media (min-width: size.$breakpoint-m) {
+  .c-comment-form__reply-buttons {
```

Comment:
This is great, but could we name it `reply-actions` so it doesn't suggest a specific element? I know, super nitpicky, it just makes it a _tad_ more general.